site/hxapi/hotel/bkg/index.md

```diff
@@ -2,205 +2,347 @@
- | SUPPLEMENT_CODE2 | 1                                 | 1                                   | 0                                | PER PERSON       | Booking Basis is PP so Adults and Children params are used, Count param is ignored        | 
- | SUPPLEMENT_CODE3 | 0                                 | 0                                   | 1                                | PER BOOKING      | Booking Basis is PB so Adults and Children params are used, Count param is ignored        | 
- | SUPPLEMENT_CODE4 | 0                                 | 0                                   | 1                                | PER ROOM         | Booking Basis is PP so Count param is used and the Adults and Children params are ignored | 
- | SUPPLEMENT_CODE5 | 0                                 | 0                                   | 2                                | PER ROOM         | Booking Basis is PP so Count param is used and the Adults and Children params are ignored | 
-
-##### The same data as above expressed in the way that it should be submitted
-
- | Field                                   | Value              | 
- | -----                                   | -----              | 
- | Supplements[]                           | 'SUPPLEMENT_CODE1' | 
- | Supplements[]                           | 'SUPPLEMENT_CODE2' | 
- | Supplements[]                           | 'SUPPLEMENT_CODE3' | 
- | Supplements[]                           | 'SUPPLEMENT_CODE4' | 
- | Supplements[]                           | 'SUPPLEMENT_CODE5' | 
- | Field                                   | Value              | 
- | SupplementsAdults['SUPPLEMENT_CODE1']   | 1                  | 
- | SupplementsAdults['SUPPLEMENT_CODE2']   | 1                  | 
- | SupplementsAdults['SUPPLEMENT_CODE3']   | 0                  | 
- | SupplementsAdults['SUPPLEMENT_CODE4']   | 0                  | 
- | SupplementsAdults['SUPPLEMENT_CODE5']   | 0                  | 
- | Field                                   | Value              | 
- | SupplementsChildren['SUPPLEMENT_CODE1'] | 0                  | 
- | SupplementsChildren['SUPPLEMENT_CODE2'] | 1                  | 
- | SupplementsChildren['SUPPLEMENT_CODE3'] | 0                  | 
- | SupplementsChildren['SUPPLEMENT_CODE4'] | 0                  | 
- | SupplementsChildren['SUPPLEMENT_CODE5'] | 0                  | 
- | Field                                   | Value              | 
- | SupplementsCount['SUPPLEMENT_CODE1']    | 0                  | 
- | SupplementsCount['SUPPLEMENT_CODE2']    | 0                  | 
- | SupplementsCount['SUPPLEMENT_CODE3']    | 1                  | 
- | SupplementsCount['SUPPLEMENT_CODE4']    | 1                  | 
- | SupplementsCount['SUPPLEMENT_CODE5']    | 2                  | 
-
-### Reply
+Below are worked examples of both the request and response for booking hotels.
```

Comment:
I've included 'worked' in other parts of docs I've reviewed. Will keep it here for now for consistency :)